app/assets/stylesheets/components/_sliding_screen.css.sass

```diff
@@ -1,5 +1,4 @@
 .SlidingScreen
-  left: 0
```

Comment:
The default value auto. [Here](http://www.w3schools.com/cssref/pr_pos_left.asp).